test/unit/action/log.spec.js

```diff
@@ -97,10 +100,20 @@ describe('Action Logs Unit Tests', () => {
         sandbox.stub(console, 'error');
         log.error('foo', err);
         expect(console.error, 'was called with', 'foo', err);
-        sandbox.restore();
         expect(store.logs.length, 'to equal', 25);
         expect(ipcRenderer.send, 'was called with', 'log-error', ['foo', err]);
       });
     });
+
+    describe('error()', () => {
+      it('should call console.error with all args', () => {
+        new LogAction(store, ipc, false);
+        const err = new Error('bar');
+        sandbox.stub(console, 'error');
+        log.error('foo', err);
+        expect(console.error, 'was called with', 'foo', err);
+        expect(store.logs.length, 'to equal', 56);
```

Comment:
Is it `56` because that's the length of the stringified `Error('bar')`?

Reply:
just a diff str length to be checked